collections/_article/year-in-review-2024.md

```diff
@@ -0,0 +1,233 @@
+
+Build your own settlements in *Of Life and Land* and experience different maps, scenarios, and rich simulations. Strategize with animals and nature in mind!
+<hr>
+
+## May
+Did you know that we still update Godot 3? Not everyone upgraded their projects when the big codebase refresh happened last year, so by offering two versions alongside each other we aim to accommodate these existing games as well as lower-end systems used by developers.
+
+The feature freeze for 3.6 happened in May, leading up to the most recent [release in September](https://godotengine.org/article/godot-3-6-finally-released/).
+
+Look at these shiny highlights:
+- [2D Physics Interpolation](https://godotengine.org/article/godot-3-6-finally-released/#2d-physics-interpolation-gh-76252)
+- [2D Hierarchical Culling](https://godotengine.org/article/godot-3-6-finally-released/#2d-hierarchical-culling-gh-68738) 
+- [Tighter Shadow Culling](https://godotengine.org/article/godot-3-6-finally-released/#tighter-shadow-culling-gh-84745) 
+- [Mesh Merging](https://godotengine.org/article/godot-3-6-finally-released/#mesh-merging-gh-61568) 
+- [Discrete Level of Detail (LOD)](https://godotengine.org/article/godot-3-6-finally-released/#discrete-level-of-detail-lod-gh-85437) 
+- [ORM Materials](https://godotengine.org/article/godot-3-6-finally-released/#orm-materials-gh-76023)
+
+<hr>
+[Arctic Eggs](https://store.steampowered.com/app/2763670/Arctic_Eggs) 
+
+This game about cooking eggs in a cold climate found great reception on the internet. There is just something about the frustratingly fun mechanics of *Arctic Eggs* that captured quite an audience - who are we to judge your taste?
+<hr>
+
+## June
+Our annual community poll rolled around in June. We want to know who our users are, so we can learn about common needs and offer the best support possible. 
+
+[This year’s poll](https://docs.google.com/forms/d/1eicOppRQG2RFZ8CjIGFf5Kha5yklO854fV8-YFtlWxk/viewanalytics) had more responses than ever before, once again proving the incredible growth the Godot community is experiencing.
+
+<img alt="poll results: programming languages" src="/storage/blog/2024-review/poll1.jpg"/>
+<img alt="poll results: donations" src="/storage/blog/2024-review/poll2.jpg"/>
+
+<hr>
+[Until Then](https://store.steampowered.com/app/1574820/Until_Then)
+
+Nominated for a plethora of awards, the visual novel *Until Then* tells the emotional story of the highschooler Mark in polished pixel art. Build relationships, play minigames, use social media, and most importantly: solve the mystery at hand.
+<hr>
+
+## July
+The [Godot robot plushies](https://www.makeship.com/products/godot-robot-plush) took over the internet in a storm. In a collaboration with Makeship, we finally proved our infamous logo’s worth. Toothy grin and googly eyes translated adorably into the 3D medium. By the end of the campaign, almost 4.000 cuddly coding companions were headed towards new homes. 
```

Comment:
```suggestion
The [Godot robot plushies](https://www.makeship.com/products/godot-robot-plush) took the internet by storm. In a collaboration with Makeship, we finally proved our infamous logo’s worth. Toothy grin and googly eyes translated adorably into the 3D medium. By the end of the campaign, almost 4.000 cuddly coding companions were headed towards new homes. 
```
"took X over in a storm" is not a common expression but "took X by storm" is. Not sure if that's what you were going for here?
```suggestion
The [Godot robot plushies](https://www.makeship.com/products/godot-robot-plush) took the internet by storm. In a collaboration with Makeship, we finally proved our infamous logo’s worth. Toothy grin and googly eyes translated adorably into the 3D medium. By the end of the campaign, almost 4,000 cuddly coding companions were headed towards new homes. 
```
The rest of this article also uses `,` for the thousands separator

Reply:
Thank you!